apps/docs/src/components/Dashboard/Usage.tsx

```diff
@@ -0,0 +1,106 @@
+import { useEffect, useState } from 'react'
+import { Card, CardContent } from '../ui/card'
+import { LoaderIcon } from 'lucide-react'
+import LineChart from './Chart'
+
+const usageUrl = `${process.env.NEXT_PUBLIC_BILLING_API_URL}/teams/usage`
+
+export const UsageContent = ({ currentApiKey }: { currentApiKey: string | null}) => {
+  const [vcpuData, setVcpuData] = useState<any>(null)
+  const [ramData, setRamData] = useState<any>(null)
+
+  useEffect(() => {
+    const getUsage = async () => {
+      setVcpuData(null)
+      setRamData(null)
+
+      const response = await fetch(usageUrl, {
+        headers: {
+          'X-Team-API-Key': currentApiKey! 
```

Comment:
you can pass it as parameter and you won't need `!`

Reply:
ah that's smooth, thanks!